From 40f25064e2ab1101483f07f1bb40768b582e6122 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 25 Sep 2024 11:14:19 +0800
Subject: [PATCH 1/3] eurasiacon: kbuild: Makefile.template: replace .SECONDARY
 with .NOTINTERMEDIATE

Make 4.4 introduced .NOTINTERMEDIATE, and it appears in .FEATURES. Replace
.SECONDARY with .NOTINTERMEDIATE in newer Make version to fix below building issue:

| make[5]: *** .NOTINTERMEDIATE and .SECONDARY are mutually exclusive.  Stop.
| make[4]: *** [work-shared/ti-am335x/kernel-source/Makefile:1921: ti-sgx-ddk-km/1.17.4948957/git/eurasiacon
/binary_ti335x_linux_lws-generic_release/target_armel/kbuild] Error 2
| make[3]: *** [Makefile:234: __sub-make] Error 2

(LOCAL REV: NOT UPSTREAM) - Wrlinux specific

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 eurasiacon/build/linux2/kbuild/Makefile.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eurasiacon/build/linux2/kbuild/Makefile.template b/eurasiacon/build/linux2/kbuild/Makefile.template
index 822c3df..02bd886 100644
--- a/eurasiacon/build/linux2/kbuild/Makefile.template
+++ b/eurasiacon/build/linux2/kbuild/Makefile.template
@@ -54,7 +54,7 @@
 
 include $(OUT)/config_kernel.mk
 
-.SECONDARY:
+.NOTINTERMEDIATE:
 
 $(OUT)/$(TARGET_PRIMARY_ARCH)/kbuild/external/%.c: $(EXTRA_KBUILD_SOURCE)/%.c
 	@if [ ! -e $(dir $@) ]; then mkdir -p $(dir $@); fi
-- 
2.34.1

